src/ui/containers/book-search.container.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import SearchBox from './../components/search-box';
+
+const mapStateToProps = createStructuredSelector({
+  watermark: () => 'watermark',
+});
+
+const mapDispatchToProps = {
+  onSubmit: () => ({ type: 'SUBMIT!' }),
+  onChange: () => ({ type: 'CHANGE' }),
+};
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps,
+)(SearchBox);
```

Comment:
This is a "Connected" component. Meaning it is aware of the redux store.  It's composed of dumb components and injects actions and state into them.

- `mapStateToProps` pull values _from_ the store - in this case it's just hardcoded text.
- `mapDispatchToProps` enhances the dumb components with actions calls so the components can speak _to_ the store.